src/Certify.Shared/Utils/RdapService.cs

```diff
@@ -59,7 +59,7 @@ public class RdapService
 
         public async Task Init()
         {
-            var dat = System.IO.File.ReadAllText("Assets\\public_suffix_list.dat");
+            var dat = System.IO.File.ReadAllText("Assets/public_suffix_list.dat");
```

Comment:
Let's try to standardise on Path.Combine instead of trading specific path separators